recipes/deepdirect/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "deepdirect" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deepdirect-{{ version }}.tar.gz
+  sha256: 10feba8b3b52fbb274bdd0bdf9e8ae75df6669029d359a2ba9fae51af53a2faa
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('deepdirect', max_pin="x.x") }}
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python
+    - tensorflow >=2.4.0
+    - keras >=2.4.0
+    - colorlog ==6.7.0
+    - tqdm >=4.64.1
+
+test:
+  imports:
+    - deepdirect
+  commands:
+    - deepdirect -v
+
+about:
+  home: https://github.com/Jappy0/deepdirect
+  summary: In silico approach to generate mutations for protein complexes towards a specified direction (increase/decrease) in binding affinity.
+  license: GPL-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - Tian Lan and Pengyao Ping
```

Comment:
Please use GitHub username so that you can be notified for review when the bot automatically updates or bumps the package

Reply:
Hi @thanhleviet,

Thanks so much. I have modified the maintainer to my GitHub username. Please let me know if there are any other issues with my recipe.